core/backend/src/rpc-impl/RpcBriefcaseUtility.ts

```diff
@@ -171,7 +171,7 @@ export class RpcBriefcaseUtility {
       db = await SnapshotDb.openCheckpointV2(checkpoint);
       Logger.logTrace(loggerCategory, "using V2 checkpoint briefcase", () => ({ ...tokenProps }));
     } catch (e) {
-      Logger.logTrace(loggerCategory, "unable to open V2 checkpoint - falling back to V1 checkpoint", () => ({ ...tokenProps }));
+      Logger.logTrace(loggerCategory, "unable to open V2 checkpoint - falling back to V1 checkpoint", () => ({ error: e, ...tokenProps }));
```

Comment:
Actually, @nick4598 can we change this to use `BentleyError.getErrorProps(err)`?  Sorry I **just** thought of that after I approved. 🤦‍♂️